CMakeLists.txt

```diff
@@ -199,7 +199,7 @@ endif()
 
 # ---[ Build flags
 set(CMAKE_C_STANDARD 99)
-set(CMAKE_CXX_STANDARD 11)
```

Comment:
This will need to likely match a change in internal buck TARGETS scripts.

Reply:
are we moving to C++14 as a requirement? cc: @ezyang